src/main/scala/mesosphere/marathon/state/AppDefinition.scala

```diff
@@ -274,11 +274,12 @@ case class AppDefinition(
     )
   }
 
-  private val portIndicies: Range = container.flatMap(_.hostPorts).getOrElse(portNumbers).indices
+  private val portIndicies: Range = container.flatMap(_.hostPorts.filter(_.nonEmpty)).getOrElse(portNumbers).indices
```

Comment:
Doh!
On Tue, Aug 9, 2016 at 12:16 PM Gastón Kleiman <notifications@github.com>
wrote:

> In src/main/scala/mesosphere/marathon/state/AppDefinition.scala
> <https://github.com/mesosphere/marathon/pull/4199#discussion_r74126234>:
>
> > @@ -274,11 +274,12 @@ case class AppDefinition(
> >      )
> >    }
> >
> > -  private val portIndicies: Range = container.flatMap(_.hostPorts).getOrElse(portNumbers).indices
> > +  private val portIndicies: Range = container.flatMap(_.hostPorts.filter(_.nonEmpty)).getOrElse(portNumbers).indices
>
> nit: it should be portIndices =)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mesosphere/marathon/pull/4199/files/37309dddf919cc7501afc364c2c4f007317d51e7#r74126234>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAweZiqELL7WHjfBV7PafIWVw-ggnr06ks5qeNIRgaJpZM4Jga-X>
> .
>